src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -539,31 +576,19 @@ int encrypt(byte[] in, int inOfs, int inLen, byte[] out, int outOfs) {
             ibuffer.write(in, inOfs + inLen, remainder);
         }
 
+        // Encrypt the remaining blocks inside of 'in'
         if (len > 0) {
```

Comment:
Shouldn't this be 'inLen' instead of 'len'?

Reply:
Yes, that appears to be right